howtos/add-new-repository.md

```diff
@@ -0,0 +1,13 @@
+
+# Add a new repository to Eiffel Community
+
+Want to extend the Eiffel community with another project?
+
+1. [Contact the community](../CONTACT.md) about the new repository you want to create, to see if it's a good fit.
```

Comment:
This link target doesn't really say where to send such a proposal. It only states contact aliases for all existing repos.
I don't know what would be the best way to contact the community for such a request. Maybe Slack?

Reply:
Another option I had in mind was to create an issue in the community repository to have a discussion around it. What do you think? I'll add the Slack option.

Reply:
I thought of that possibility as well, and maybe it's good. I just think that it would not be an issue that is _solved_ in the community repo itself, i.e. no PR will be connected to it. But maybe that is not an issue (pun intended)?

Reply:
I updated with the option to also create an issue for such questions. In my opinion I don't see the problem of "solving" the issue without a linked PR. 

I think an open issue could be a good place to ask further questions if needed, and then close the issue when we reach an agreement, or when the actual repository has been created.

Reply:
I agree